migration/rules/src/main/scala/fix/v0_9_28.scala

```diff
@@ -0,0 +1,22 @@
+package fix
+
+import scala.meta._
+
+import scalafix.v1._
+
+class v0_9_28 extends SemanticRule("v0_9_28") {
+  override def fix(implicit doc: SemanticDocument): Patch =
+    doc.tree.collect {
+      case t @ Importee.Name(Name("SemanticRuleSuite")) =>
+        Patch.removeImportee(t) +
+          Patch.addGlobalImport(
+            importer"scalafix.testkit.AbstractSemanticRuleSuite"
+          )
+      case t @ Init(_, _, _) =>
+        if (t.toString() == "SemanticRuleSuite()")
```

Comment:
use semanticdb information through [`SymbolMatcher`](https://scalacenter.github.io/scalafix/docs/developers/symbol-matcher.html) rather than relying on representation (the empty parameters list being optional, this will yield false negatives).